examples/access-mutate-java-fields/access-mutate-java-fields.description

```diff
@@ -0,0 +1,3 @@
+Java fields (both static and instance) can be read and modified from Ballerina.
```

Comment:
The .description file name should be the same as the folder name, with only the `-`s replaced with `_`s.

e.g., access_mutate_java_fields.description

Shall we also update the .bal and .out similarly?